flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/NFAState.java

```diff
@@ -97,6 +100,18 @@ public void setNewPartialMatches(PriorityQueue<ComputationState> newPartialMatch
         this.partialMatches = newPartialMatches;
     }
 
+    public boolean isNewStartPartialMatch() {
+        return isNewStartPartialMatch;
+    }
+
+    public void resetNewStartPartialMatch() {
```

Comment:
Do the `resetNewStartPartialMatch`  and `setNewStartPartiailMatch` combine to one method?

Reply:
I refer to the implementation style of `resetStateChanged`\`setStateChanged` in NFAState here. I think both worked